src/arrayMethodSplice.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Splice
  */
 function applyCustomSplice() {
-  [].__proto__.splice2 = function(start, deleteCount, ...items) {
-    // write code here
+  [].__proto__.splice2 = function(
+    start = 0, deleteCount = this.length, ...items) {
```

Comment:
if you have too long line with arguments, place each on the separate line
```
[].__proto__.splice2 = function(
    start = 0, 
    deleteCount = this.length, 
    ...items
  ) {...
```

Reply:
> if you have too long line with arguments, place each on the separate line
> 
> ```
> [].__proto__.splice2 = function(
>     start = 0, 
>     deleteCount = this.length, 
>     ...items
>   ) {...
> ```

fixed